### PR TITLE
docs: add standalone details to NG8001

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
improve the degubbing section of the NG8001 error by including details
about standalone components

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Screenshot at 2022-05-20 22-01-17

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
 - @AndrewKushnir

### CODE REVIEW

File: aio/content/errors/NG8001.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ This is the compiler equivalent of a common runtime error `NG0304: '${tagName}'
 Use the element name in the error to find the file(s) where the element is being used.
 
 Check that the name and selector are correct.
-If the component is from a different module or import, check that the component is exported from its origin module and imported into the correct `*.modules.ts` file, and declared in the imports list.
+
+Make sure that the component is correctly imported inside your module or standalone component (by checking its presence in the `imports` field), if your component is declared in a module (meaning that it is not standalone) make sure that it is exported correctly from it.
```
Comment (AndrewKushnir) [inline aio/content/errors/NG8001.md:19]:
```suggestion
Make sure that the component is correctly imported inside your NgModule or standalone Component, by checking its presence in the `imports` field. If the component is declared in an NgModule (meaning that it is not standalone) make sure that it is exported correctly from it, by checking its presence in the `exports` field.
```

**Review by AndrewKushnir — COMMENTED** [review]
@dario-piotrowicz looks great, just a few minor proposals.

**Comment by AndrewKushnir** [comment]
> I was tempted to add something around that but it may end up being a bit of a longer explanation or a more concise but cryptic one (I assume that the info here should be as simple and as short as possible)...

I think the description that you have looks good and we can improve it more in the future once we get additional feedback. Thank you.

File: aio/content/errors/NG8001.md (hunk 1 above)
Comment (dario-piotrowicz) [reply aio/content/errors/NG8001.md]: @AndrewKushnir I've applied your suggested change, but I just noticed that you capitalized only one instance of "component" but not the others, was that intentional?

(by the way I randomly checked a couple of pages on aio and it seems like component is often kept lowercase)

Comment (AndrewKushnir) [reply aio/content/errors/NG8001.md]: Yeah, good point, let's lowercase it here as well 👍

Comment (dario-piotrowicz) [reply aio/content/errors/NG8001.md]: updated :slightly_smiling_face:

**Review by AndrewKushnir — APPROVED** [review]

**Review by pkozlowski-opensource — APPROVED** [review]

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: public-api

**Review by jessicajaniuk — APPROVED** [review]
LGTM :cookie:

reviewed-for: public-api

**Comment by alxhub** [comment]
This PR was merged into the repository by commit a43385cca4ee1cbde716a1f20c4444f844a5d46b.

**Pull request closed without merging**